Remotion/Web/Core/ContentSecurityPolicy/CspDefaultsProvider.cs

```diff
@@ -0,0 +1,59 @@
+﻿// SPDX-FileCopyrightText: (c) RUBICON IT GmbH, www.rubicon.eu
+// SPDX-License-Identifier: LGPL-2.1-or-later
+using System.Web.UI;
+using Remotion.ServiceLocation;
+using Remotion.Utilities;
+
+namespace Remotion.Web.ContentSecurityPolicy;
+
+/// <summary>
+/// Implements <see cref="ICspDefaultsProvider"/> to provide a default service with opt-in CSP support with a default header.
+/// Register a custom instance to enable CSP and set a custom CSP header.
+/// </summary>
+/// <threadsafety static="true" instance="true" />
+[ImplementationFor(typeof(ICspDefaultsProvider), Lifetime = LifetimeKind.Singleton)]
```

Comment:
please add a test fixture `ICspDefaultsProviderTest` for testing the correct IoC configuration.